content/partials/pricing.html

```diff
@@ -33,6 +33,12 @@ <h2 class="ui small monospace center aligned header">
 
       {{ caller() }}
 
+        {% if highlight %}
+        <div class="ui teal bottom attached label">
+          Most popular
```

Comment:
I'd make this at least more wordy.

```suggestion
          Our most popular plan
```

You might also add an icon to improve the visibility of this though.